Foundation/Operation.swift

```diff
@@ -234,81 +234,163 @@ extension OperationQueue {
     public static let defaultMaxConcurrentOperationCount: Int = Int.max
 }
 
+internal class _IndexedOperationLinkedList {
+    
+    internal class Node {
+        var operation: Operation
+        var next: _IndexedOperationLinkedList.Node? = nil
+        weak var previous: _IndexedOperationLinkedList.Node? = nil
+        
+        init(operation: Operation) {
+            self.operation = operation
+        }
+    }
+    
+    private(set) var root: _IndexedOperationLinkedList.Node? = nil
+    private(set) weak var tail: _IndexedOperationLinkedList.Node? = nil
+    private(set) var count: Int = 0
+    private var nodeForOperation: [Operation: _IndexedOperationLinkedList.Node] = [:]
+    
+    func insert(_ operation: Operation) {
```

Comment:
I would really call this append to avoid ambiguity

Reply:
Consider it done.

Reply:
Resolving this comment since it is addressed